app/src/main/java/de/danoeh/antennapod/fragment/preferences/UserInterfacePreferencesFragment.java

```diff
@@ -71,7 +76,12 @@ private void setupInterfaceScreen() {
                     }
                     final Context context = getActivity();
                     final String[] navTitles = context.getResources().getStringArray(R.array.back_button_go_to_pages);
-                    final String[] navTags = context.getResources().getStringArray(R.array.back_button_go_to_pages_tags);
+                    final String[] navTags = new String[]{
+                            InboxFragment.TAG,
+                            QueueFragment.TAG,
+                            EpisodesFragment.TAG,
+                            SubscriptionFragment.TAG
+                    };
```

Comment:
This looks like it can easily get out of sync. Can't you still use the array?

Reply:
Hmm not really sure what I did there, I think in general it's easier to have the arrays in code instead of in strings.xml (because then it's easy to get out of sync)